spec/cypress/e2e/annotations_overview_spec.cy.js

```diff
@@ -0,0 +1,240 @@
+      { medium_id: this.medium3.id, user_id: this.user.id,
+        comment: "This is a math annotation: $\\frac{1}{2}$",
+      }).as("annotationWithMath");
+
+    cy.then(() => {
+      cy.visit("/annotations/overview");
+      cy.getBySelector(CARD_SELECTOR).last().then(($card) => {
+        cy.wrap($card).find(".katex").should("exist");
+      });
+    });
+  });
+
+  it("contains medium and annotation information", function () {
+    cy.visit("/annotations/overview");
+    [
+      { title: MEDIUM_TITLE_1, annotation: this.annotation1, lesson: this.lesson1 },
+      { title: MEDIUM_TITLE_2, annotation: this.annotation2, lesson: this.lesson2 },
+      { title: MEDIUM_TITLE_3, annotation: this.annotation3, lesson: this.lesson3 },
+      { title: MEDIUM_TITLE_3, annotation: this.annotation4, lesson: this.lesson3 },
+    ].forEach((test, i) => {
+      cy.getBySelector(CARD_SELECTOR).eq(i).as("card");
+
+      // Lesson date
+      const expectedDate = this.user.locale === "de"
+        ? new Date(test.lesson.date).toLocaleDateString("de-DE", {
+          year: "numeric",
+          month: "2-digit",
+          day: "2-digit",
+        })
+        : test.lesson.date;
+      cy.get("@card").children().first().should("contain", expectedDate);
+
+      // Medium title
+      cy.get("@card").children().first().should("contain", test.title);
+
+      // Annotation category & comment
+      cy.get("@card").children().first().invoke("text").then((categoryText) => {
+        expect(categoryText.toLowerCase()).to.contain(test.annotation.category);
+      });
+      cy.get("@card").children().eq(1).should("contain", test.annotation.comment);
+    });
+  });
+
+  it("has border according to annotation color", function () {
+    cy.visit("/annotations/overview");
+    [this.annotation1, this.annotation2, this.annotation3, this.annotation4]
+      .forEach((annotation, i) => {
+        cy.getBySelector(CARD_SELECTOR).eq(i).as("card");
+        const colorExpected = hexToRgb(annotation.color);
+        cy.get("@card").should("have.css", "border-color", colorExpected);
+      });
+  });
+
+  it("redirects to medium video when clicked", function () {
+    [
+      { medium: this.medium1, annotation: this.annotation1 },
+      { medium: this.medium2, annotation: this.annotation2 },
+      { medium: this.medium3, annotation: this.annotation3 },
+      { medium: this.medium3, annotation: this.annotation4 },
+    ].forEach((test, i) => {
+      cy.visit("/annotations/overview");
+      cy.getBySelector(CARD_SELECTOR).eq(i).as("card");
+      cy.get("@card").parents(".accordion-collapse").siblings(".accordion-header").click();
+      cy.get("@card").click();
+
+      cy.url().should("contain", `/media/${test.medium.id}`);
+      let timestamp = `0:00:${test.annotation.timestamp.seconds}`;
+      cy.getBySelector("current-time").should("contain", timestamp);
+      cy.getBySelector("annotation-comment").should("contain", test.annotation.comment);
```

Comment:
This fails in my setup.
![Screenshot 2024-08-12 113157](https://github.com/user-attachments/assets/d187875b-383b-403e-ad09-71b0cd667b25)